.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
     rev: v2.4.0
     hooks:
       - id: prettier
-        args: [--write, ./frontend]
+        args: [--write]
```

Comment:
remove `./frontend`, so prettier will only lint staged files instead of linting the whole frontend directory